web/src/app/admin/connectors/[connector]/AddConnectorPage.tsx

```diff
@@ -203,6 +265,7 @@ export default function AddConnector({
             advancedConfiguration.refreshFreq,
             advancedConfiguration.pruneFreq,
             advancedConfiguration.indexingStart,
+            values.access_type == "public",
```

Comment:
Nit: if access type is no longer a boolean would prefer all functions get update to move handling of access type within them (less room for errors long term.) Ie. change arg type to Access Type and get rid of these conversions to bools

Reply:
Saving this for connector rework